scripts/linters/pylint_extensions.py

```diff
@@ -1772,45 +1772,37 @@ class BlankLineBelowFileOverviewChecker(checkers.BaseChecker):
         )
     }
 
-    def process_module(self, node):
-        """Process a module to ensure that there is a blank line below
+    def visit_module(self, node):
+        """Visit a module to ensure that there is a blank line below
         file overview docstring.
 
         Args:
             node: astroid.scoped_nodes.Function. Node to access module content.
         """
-
-        multi_line_indicator = b'"""'
-        file_content = read_from_node(node)
-        file_length = len(file_content)
-        triple_quote_counter = 0
-        empty_line_counter = 0
-        for line_num in python_utils.RANGE(file_length):
-            line = file_content[line_num].strip()
-            # Single line comment, ignore it.
-            if line.startswith(b'#'):
-                continue
-            triple_quote_counter += line.count(multi_line_indicator)
-
-            if line.endswith(b'"""') and triple_quote_counter == 2:
-                closing_line_index_of_fileoverview = line_num
+        # Check if the given node has docstring.
+        if node.doc is None:
+            return
+        line_number = node.fromlineno
+        # Iterate till the start of docstring.
+        while True:
+            line = linecache.getline(node.root().file, line_number).strip()
+            if line.startswith((b'"""', b'\'\'\'', b'\'', b'"')):
```

Comment:
This check is weird. Why do we have a check testing `"""` or `"`? The first one would be automatically covered in the later part.

Also, what if we have a string that gets captured by this check and is not a docstring?

Reply:
Done. No the string will not be captured by this, because there is nothing above file overview except comments.